jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -997,6 +997,17 @@ public static void load(String pathName) {
  */
 @CallerSensitive
 public static void loadLibrary(String libName) {
+	if (libName.indexOf(File.pathSeparator) >= 0) {
+		/*[MSG "K0B01", "Library name must not contain a file path: {0}]*/
+		throw new UnsatisfiedLinkError(com.ibm.oti.util.Msg.getString("K0B01", libName)); //$NON-NLS-1$
+	}
+	if (System.internalGetProperties().getProperty("os.name").startsWith("Windows")) { //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
```suggestion
	if (systemProperties.getProperty("os.name").startsWith("Windows")) { //$NON-NLS-1$ //$NON-NLS-2$
```

Reply:
The qualification could be removed (`internalGetProperties()` instead of `System.internalGetProperties()`), but removing the call would bypass the check of `propertiesInitialized`.